src/common/Types.ts

```diff
@@ -1148,9 +1149,15 @@ export interface ClipboardEvent extends SyntheticEvent {
 
 export type FocusEvent = SyntheticEvent;
 
+export enum MouseButton {
+    Primary,
+    Auxiliary,
+    Secondary
+}
+
 export interface MouseEvent extends SyntheticEvent {
     altKey: boolean;
-    button: number;
+    button: MouseButton;
```

Comment:
This is a breaking interface change. While I like the use of an enumeration here, it's not worth the pain associated with breaking existing code. Please revert back to "number".